src/components/card/z-card-footer/index.tsx

```diff
@@ -57,7 +57,9 @@ export class ZCardFooter {
               </span>
             </p>
           </div>
-          {this.isOpen && <slot name="list" />}
+          <div class={`slot-handler ${this.isOpen ? 'visible' : 'nonvisible'}`}>
```

Comment:
Maybe `hidden` instead of `nonvisible`? :sweat_smile:

Reply:
resolved with commit 7ea2d681eddb424b9a02f324eaa5e4cecd6bb064